docs/Common/Enums/index.md

```diff
@@ -802,9 +878,57 @@
 |OFF|1||
 |ON|2||
 
-### EntityStatus
+### ModuleType
+
+|Name|Value|Description|
+|:---|:----|:----------|
+|CLIMATE|0||
+|RADIO|1||
+
+### RadioBand
+
+|Name|Value|Description|
+|:---|:----|:----------|
+|AM|0||
+|FM|1||
+|XM|2||
+
+### RadioState
+
+|Name|Value|Description|
+|:---|:----|:----------|
+|ACQUIRING|0||
+|ACQUIRED|1||
+|MULTICAST|2||
+|NOT_FOUND|3||
+
+### TemperatureUnit
+
 |Name|Value|Description|
 |:---|:----|:----------|
-|ON|||
-|OFF||| 
+|FAHRENHEIT|0||
+|CELSIUS|1||
+
+### DefrostZone
+
+|Name|Value|Description|
+|:---|:----|:----------|
+|FRONT|0||
+|REAR|1||
+|ALL|2||
+|NONE|3||
 
+### VentilationMode
+
+|Name|Value|Description|
+|:---|:----|:----------|
+|UPPER|0||
+|LOWER|1||
+|BOTH|2||
+|NONE|3||
+
+### EntityStatus
```

Comment:
This should've been part of 4.3.0

Reply:
NVM, the diff says that it was added and removed, it just formatted this part oddly